requirements/base.txt

```diff
@@ -6,6 +6,7 @@ whitenoise==4.1.2  # https://github.com/evansd/whitenoise
 redis==3.2.1  # https://github.com/antirez/redis
 celery==4.3.0  # pyup: < 5.0  # https://github.com/celery/celery
 flower==0.9.3  # https://github.com/mher/flower
+vine==1.3.0
```

Comment:
Was this meant to be added?

Reply:
Yeah, our build was failing because pytest is looking for a module in vine that doesn't exist in the version it uses as a dependency, so I'm just pinning it to the version we want here.

Reply:
Ok sounds good. Do you mind adding a comment regarding the min version we require and why?